templates/v-core-3-printer.template.cfg

```diff
@@ -140,6 +140,11 @@ pid_Kd: 114
 [include config/macros.cfg]
 [include config/printers/v-core-3/macros.cfg]
 
+### DISPLAY MENUS
+#[include config/menus/calibration.cfg]
+#[include config/menus/filament-change.cfg]
+#[include config/menus/printer.cfg]
+
```

Comment:
I think a config/menus/*.cfg would be good here!

Reply:
Moved and updated though we might have to change the wildcard back in the future once we add more optional menus.